src/app/core/token-interceptor.service.ts

```diff
@@ -0,0 +1,24 @@
+import {Inject, Injectable} from '@angular/core';
+import {HttpEvent, HttpHandler, HttpRequest} from '@angular/common/http';
+import {Observable} from 'rxjs';
+import {AuthConfig, AuthTokenConfig} from '@core';
+
+@Injectable()
+export class TokenInterceptorService {
+
+  constructor(@Inject(AuthConfig) private config: AuthTokenConfig) {
+  }
+
+  intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+    const token = localStorage.getItem('app_token');
```

Comment:
Do not read `localStorage` here. Extend AuthService to store this value inside a public property instead.